working-with-collections/main.js

```diff
@@ -0,0 +1,67 @@
+/* global _ */
+/* eslint-disable no-console */
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+setTimeout(() => { console.log('Type "play()" in console for A New Game'); }, 500);
+
+const cardsSuit = ['clubs', 'diamonds', 'hearts', 'spades'];
+const cardsRank = ['A', 2, 3, 4, 5, 6, 7, 8, 9, 10, 'J', 'Q', 'K'];
+const cardsData = [];
+const player = ['Annie', 'Tim D.', 'Cody', 'Uzair'];
+const cardsPerHand = 2;
+
+function createCards(suit, rank) {
+  cardsSuit.forEach(x => {
+    cardsRank.forEach(y => {
+      if (y === 'A') {
+        cardsData.push({ card: `${x} ${y}`, value: 11 });
+      } else if (y === 'J' || y === 'Q' || y === 'K') {
+        cardsData.push({ card: `${x} ${y}`, value: 10 });
+      } else {
+        cardsData.push({ card: `${x} ${y}`, value: y });
+      }
+    });
+  });
+  _.shuffle(cardsData);
```

Comment:
`_.shuffle` does not modify the original array, but it returns a shuffled copy

https://lodash.com/docs/4.17.15#shuffle